charts/incubator/palworld/values.yaml

```diff
@@ -26,6 +26,10 @@ service:
 palworld:
   game:
     id: "2394010"
+    name: "Palworld Docker"
+    description: "Palworld running in Docker"
```

Comment:
Its not docker ;)

Reply:
just the default values, but ill change it.